app/scripts/controllers/user-storage/encryption/encryption.ts

```diff
@@ -45,7 +54,9 @@ class EncryptorDecryptor {
     try {
       const encryptedData: EncryptedPayload = JSON.parse(encryptedDataStr);
       if (encryptedData.v === '1') {
-        return this.#decryptStringV1(encryptedData, password);
+        if (encryptedData.t === 'scrypt') {
+          return this.#decryptStringV1(encryptedData, password);
+        }
       }
       throw new Error(`Unsupported encrypted data payload - ${encryptedData}`);
```

Comment:
The error string is stringfied to:

```
Unable to decrypt string - Unsupported encrypted data payload - [object Object]
```

We can either log the `encryptedDataStr` string or omit it completely.

Reply:
Great spot. Will replace with `encryptedDataStr` :+1: